azext_edge/edge/_help.py

```diff
@@ -780,6 +780,681 @@ def load_iotops_help():
+          text: >
+            az iot ops dataflow endpoint create local-mqtt
+            --name myendpoint
+            --instance mycluster-ops-instance
+            --resource-group myresourcegroup
+            --hostname aio-broker
+            --port 1883
+            --secret-name mysecret
+        - name: Create or replace a dataflow endpoint resource anonymously.
+          text: >
+            az iot ops dataflow endpoint create local-mqtt
+            --name myendpoint
+            --instance mycluster-ops-instance
+            --resource-group myresourcegroup
+            --hostname aio-broker
+            --port 1883
+            --no-auth
+        - name: Show config file for creating a dataflow endpoint resource.
+          text: >
+            az iot ops dataflow endpoint create local-mqtt
+            --name myendpoint
+            --instance mycluster-ops-instance
+            --resource-group myresourcegroup
+            --hostname aio-broker
+            --port 1883
+            --client-id-pre myclientprefix
+            --keep-alive 100
+            --max-inflight-msg 70
+            --protocol WebSockets
+            --qos 2
+            --retain Never
+            --show-config
+    """
+
+    helps[
+        "iot ops dataflow endpoint create localstorage"
+    ] = """
+        type: command
+        short-summary: Create or replace a local storage data flow endpoint.
+
+        examples:
+        - name: Create or replace a dataflow endpoint resource with minimum input.
+          text: >
+            az iot ops dataflow endpoint create localstorage
+            --name myendpoint
+            --instance mycluster-ops-instance
+            --resource-group myresourcegroup
+            --pvc-ref mypvc
+        - name: Show config file for creating a dataflow endpoint resource.
+          text: >
+            az iot ops dataflow endpoint create localstorage
+            --name myendpoint
+            --instance mycluster-ops-instance
+            --resource-group myresourcegroup
+            --pvc-ref mypvc
+            --show-config
+    """
+
+    helps[
+        "iot ops dataflow endpoint update"
+    ] = """
+        type: group
+        short-summary: Update the properties of an existing dataflow endpoint resource.
+    """
+
+    helps[
+        "iot ops dataflow endpoint update adls"
+    ] = """
+        type: command
+        short-summary: Update the properties of an existing dataflow endpoint resource for Azure Data Lake Storage Gen2.
+
+        examples:
+        - name: Update the storage account name of the dataflow endpoint resource called 'myendpoint'.
+          text: >
+            az iot ops dataflow endpoint update adls
+            --name myendpoint
+            --instance mycluster-ops-instance
+            --resource-group myresourcegroup
+            --storage-account-name newstorageaccount
+        - name: Update the authentication method of the dataflow endpoint resource called 'myendpoint'.
```

Comment:
include what authentication method is going to be used, especially if it is not specified in the examples.